pig/src/main/java/org/jboss/pnc/bacon/pig/impl/documents/Deliverables.java

```diff
@@ -40,4 +40,5 @@
     private final String duplicateArtifactListName = "repository-duplicate-artifact-list.txt";
     private String nvrListName;
     private String sharedContentReport = "shared-content-report.csv";
+    private String offlinerFile;
```

Comment:
Correct me if I'm wrong, but it seems the offlinerFile is not used anywhere?

Reply:
Removed, because we don't produce repository by running the generateOfflinerManifest